src/main/java/views/AcesUp/AcesUp.flt.html

```diff
@@ -161,68 +165,138 @@
-    } else if (val.value == 12) {
-        $( '#c3 .l'+key ).html("Queen of " + val.suit);
-    } else if (val.value == 13) {
-        $( '#c3 .l'+key ).html("King of " + val.suit);
-    } else if (val.value == 14) {
-        $( '#c3 .l'+key ).html("Ace of " + val.suit);
-    }
-    });
+if(userChoice == "engGame")
+{
+    if(game.Score == 48)
+    window.alert("You win!")
+
+    $( '.columnOfCards .cardLocation' ).html("");
+
+        $.each(game.cols[0], function( key, val ) {
+        if (val.value < 11) {
+            $( '#c0 .l'+key ).html(val.value + " of " + val.suit);
+        } else if (val.value == 11) {
+            $( '#c0 .l'+key ).html("Jack of " + val.suit);
+        } else if (val.value == 12) {
+            $( '#c0 .l'+key ).html("Queen of " + val.suit);
+        } else if (val.value == 13) {
+            $( '#c0 .l'+key ).html("King of " + val.suit);
+        } else if (val.value == 14) {
+            $( '#c0 .l'+key ).html("Ace of " + val.suit);
+        }
+        });
+
+     $.each(game.cols[1], function( key, val ) {
+        if (val.value < 11) {
+            $( '#c1 .l'+key ).html(val.value + " of " + val.suit);
+        } else if (val.value == 11) {
+            $( '#c1 .l'+key ).html("Jack of " + val.suit);
+        } else if (val.value == 12) {
+            $( '#c1 .l'+key ).html("Queen of " + val.suit);
+        } else if (val.value == 13) {
+            $( '#c1 .l'+key ).html("King of " + val.suit);
+        } else if (val.value == 14) {
+            $( '#c1 .l'+key ).html("Ace of " + val.suit);
+        }
+        });
+
+     $.each(game.cols[2], function( key, val ) {
+        if (val.value < 11) {
+            $( '#c2 .l'+key ).html(val.value + " of " + val.suit);
+        } else if (val.value == 11) {
+            $( '#c2 .l'+key ).html("Jack of " + val.suit);
+        } else if (val.value == 12) {
+            $( '#c2 .l'+key ).html("Queen of " + val.suit);
+        } else if (val.value == 13) {
+            $( '#c2 .l'+key ).html("King of " + val.suit);
+        } else if (val.value == 14) {
+            $( '#c2 .l'+key ).html("Ace of " + val.suit);
+        }
+        });
+
+     $.each(game.cols[3], function( key, val ) {
+        if (val.value < 11) {
+            $( '#c3 .l'+key ).html(val.value + " of " + val.suit);
+        } else if (val.value == 11) {
+            $( '#c3 .l'+key ).html("Jack of " + val.suit);
+        } else if (val.value == 12) {
+            $( '#c3 .l'+key ).html("Queen of " + val.suit);
+        } else if (val.value == 13) {
+            $( '#c3 .l'+key ).html("King of " + val.suit);
+        } else if (val.value == 14) {
+            $( '#c3 .l'+key ).html("Ace of " + val.suit);
+        }
+        });
+ }
+
+ if(userChoice == "spaGame")
+{
+    if(game.Score == 36)
+    window.alert("You win!")
+
+    $( '.columnOfCards .cardLocation' ).html("");
+
+        $.each(game.cols[0], function( key, val ) {
+        if (1 < val.value < 8) {
```

Comment:
This is a syntax error. If the value is greater than 1 then this will always execute. It ignores the < 8 portion of the statement. For example if the value is 10 then this will execute.